docs/notebook_tutorial/zero_shot_text_classification.ipynb

```diff
@@ -0,0 +1,2652 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e1489859",
+   "metadata": {},
+   "source": [
+    "# Zero Shot Text Classification Example\n",
+    "\n",
+    "In this tutorial we will walk through the text classification pipeline in forte, using zero shot text classification model from hugging face. We will also explore ways to improve our model accuracy with simple modifications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c371c5c4",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "Zero shot text classification is an extreme example of transfer learning, where the model tries to predict without any fine tuning. This kind of model predict classification result based on similarity between the input text and label name, so they don't require any labeled data. Well defined Label names are sufficient to achieve a reasonably performing model. We will be using [valhalla/distilbart-mnli-12-1](https://huggingface.co/valhalla/distilbart-mnli-12-1) as our text classification model, which was trained on [Multi-nli](https://huggingface.co/datasets/multi_nli) dataset to make predictions on a subset of amazon review sentiment [ARS](https://s3.amazonaws.com/fast-ai-nlp/amazon_review_polarity_csv.tgz) dataset and [Banking77](https://raw.githubusercontent.com/PolyAI-LDN/task-specific-datasets/master/banking_data/test.csv) dataset. we will also see how the forte pipeline works seamlessly with different third party tools like [nltk](https://www.nltk.org/) and [huggingface](https://huggingface.co/) to make our life easier."
```

Comment:
Could you check grammar errors? I have seen some apparent grammar errors.

Reply:
I fixed using Grammarly.